apis/management.cattle.io/v3/alerting_types.go

```diff
@@ -300,6 +302,14 @@ type WebhookConfig struct {
 	URL string `json:"url,omitempty" norman:"required"`
 }
 
+type WechatConfig struct {
+	DefaultRecipient string `json:"defaultRecipient,omitempty" norman:"required"`
+	Secret           string `json:"secret,omitempty" norman:"required"`
```

Comment:
Shouldnt the type be "password" for this field?

Reply:
Yes, it should be. Updated. Thanks!